Makefile

```diff
@@ -67,6 +67,11 @@ test:
 test-all: export RECONCILER_INTEGRATION_TESTS = 1
 test-all: test
 
+.PHONY: test-ory
+test-ory: export ORY_RECONCILER_INTEGRATION_TESTS=1
+test-ory:
```

Comment:
Is this target duplication required? Can't we have a list of commands to be executed?

Reply:
it's target-specific variable, for the makefile this is acceptable.